streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorNodeTest.java

```diff
@@ -194,15 +193,21 @@ public void process(final Object key, final Object value) {
 
     @Test
     public void testTopologyLevelClassCastExceptionDirect() {
-        final Metrics metrics = new Metrics();
-        final StreamsMetricsImpl streamsMetrics =
-            new StreamsMetricsImpl(metrics, "test-client", StreamsConfig.METRICS_LATEST);
-        final InternalMockProcessorContext context = new InternalMockProcessorContext(streamsMetrics);
-        final ProcessorNode<Object, Object> node = new ProcessorNode<Object, Object>("name", new ClassCastProcessor(), Collections.<String>emptySet());
+        final Properties properties = StreamsTestUtils.getStreamsConfig();
+        properties.put(StreamsConfig.BUILT_IN_METRICS_VERSION_CONFIG, StreamsConfig.METRICS_LATEST);
+        final InternalProcessorContext context = new MockInternalProcessorContext(properties);
+        final ProcessorNode<Object, Object> node = new ProcessorNode<>("name", new ClassCastProcessor(), Collections.emptySet());
         node.init(context);
         final StreamsException se = assertThrows(StreamsException.class, () -> node.process("aKey", "aValue"));
         assertThat(se.getCause(), instanceOf(ClassCastException.class));
         assertThat(se.getMessage(), containsString("default Serdes"));
         assertThat(se.getMessage(), containsString("input types"));
     }
+
+    private boolean containsMetric(final StreamsMetricsImpl streamsMetrics,
+                                   final String name,
+                                   final Map<String, String> tags) {
+        final MetricName metricName = new MetricName(name, METRIC_GROUP, "", tags);
+        return streamsMetrics.metrics().containsKey(metricName);
+    }
```

Comment:
req: Add this to `StreamsTestUtils`, please. This is the method you need that I mentioned in a comment on `MockInternalProcessorContext`.